### PR TITLE
0.18.2 upgrade + any attestator can attest

### PR DESCRIPTION
**Pull request by dorin-iancu**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -7,20 +7,17 @@ mod value_state;
...
-#[elrond_wasm_derive::contract(AttestationImpl)]
+#[elrond_wasm::contract]
 pub trait Attestation {
 	#[init]
-	fn init(&self, registration_cost: &BigUint, address: Address, max_nonce_diff: u64) {
-		self.set_registration_cost(registration_cost);
-		self.set_attestator_state(&address, &ValueState::Approved);
+	fn init(&self, registration_cost: Self::BigUint, address: Address, max_nonce_diff: u64) {
+		self.registration_cost().set(&registration_cost);
+		self.attestator_state(&address).set(&ValueState::Approved);
```
Comment (sasurobert) [inline src/lib.rs]: should get var args of addresses. list of addresses.

Comment (sasurobert) [reply src/lib.rs]: and should delete old list if exist

Comment (dorin-iancu) [reply src/lib.rs]: Done.

**Review by sasurobert — APPROVED** [review]

File: tests/mandos_test.rs [hunk 2]
```diff
@@ -1,18 +1,6 @@
 extern crate attestation;
```
Comment (andrei-marinica) [inline tests/mandos_test.rs]: rename this file to mandos_test_rs.rs

File: tests/tests.rs [hunk 3]
```diff
@@ -1,18 +0,0 @@
-extern crate attestation;
```
Comment (andrei-marinica) [inline tests/tests.rs:1]: Keep this file, but rename it to mandos_go_test.rs
Also would be cool to copy the github actions from elrond-wasm.

**Review by andrei-marinica — CHANGES_REQUESTED** [review]

**Review by andrei-marinica — APPROVED** [review]

**Pull request merged**